lib/reconcile.js

```diff
@@ -138,6 +138,18 @@ module.exports = function (clusterInfo, uuid = require('uuid')) {
 
     const { common, cluster, tokens } = inputSettings
 
+    if (!cluster.credentials) {
+      try {
+        // see if we already have a SA for this project
+        const email = `${common.projectId}-k8s-sa@${common.projectId}.iam.gserviceaccount.com`
+        const creds = await clusterInfo.getServiceAccount(email)
+        if (creds) cluster.credentials = creds
+        log.info('reusing existing service account for the project')
+      } catch (e) {
+        log.error(e.stack)
```

Comment:
should we bother logging the stack here? failures caught here just mean we fallback to the common credentials, so i feel like an info log that no existing service account existed is sufficient. though i could see us wanting to assert the error is actually a not found so other classes of failure can bubble up appropriately